python/first_histogram.md

```diff
@@ -0,0 +1,242 @@
+# Making your first histogram
+
+In this section we are going to make our first plots using Python. For this we
+will use some public LHCb data for $$ B^+ \rightarrow H^+ H^+ H^- $$ from the
+[CERN open data portal](http://opendata.cern.ch/?ln=en). This data is available
+on EOS:
+
+```bash
+$ xrdfs root://eospublic.cern.ch/ ls /eos/opendata/lhcb/AntimatterMatters2017/data
```

Comment:
I see this directory (`/eos/opendata`) is not mounted on lxplus, therefore I assume the `xrdfs` command is necessary. Let me suggest you replace `xrdfs` with `eos` instead, it prints more information in this case (and it will maybe become the standard in the future, so they say).

Reply:
Ah I wasn't aware the server url could be set that way with the `eos` client, agreed! 👍